hepcrawl/utils.py

```diff
@@ -420,7 +420,7 @@ def wrapper(self, *args, **kwargs):
         if disallowed_kwargs:
             raise TypeError(
                 'Only underscored kwargs allowed in {}. '
-                'Check {} for typos.'.format(func, ', '.join(disallowed_kwargs))
+                'Check {} for typos.'.format(repr(func), repr(disallowed_kwargs))
```

Comment:
why not using the "%s"?

Reply:
Huh! I thought it was being deprecated in Python 3, TIL. I'll change it then.

Reply:
I wanted an answer more than a change ;), curious why you decided to go with this instead.

Reply:
FYI you can write `{!r}` in the format to use the `repr`.

Reply:
yep, also '%r' in the '%' style formatting.

Reply:
that is not a good solution in general though (using `{!r}`), if what you want is the string representation, in this case is enough as we are logging and just care about the object contents, but for other stuff it will print the string representation of the python object instead of the string **value** of it:
```
In [3]: print '{!r}'.format(''.join([u'é']))
u'\xe9'

In [4]: print '%s' % ''.join([u'é'])
é
```

Reply:
@david-caro ...that and `repr` seemed like an easy way out under the circumstances. Otherwise I think it would've been a mess trying to use the string representation, not knowing if it's unicode or not, and having it be python 2 & 3 compatible.